app/models/admin_section.rb

```diff
@@ -10,6 +12,8 @@ class AdminSection
     plans: 'API backends, API products',
     policy_registry: 'policies'
   }
+  SERVICE_PERMISSIONS = %i[partners plans monitoring]
```

Comment:
```suggestion
  SERVICE_PERMISSIONS = %i[partners plans monitoring].freeze
```
🙈 

But I won't request you to push again only for this 👍🏼 😉

Reply:
Wait, isn't the magic comment supposed to make this unnecessary?

Reply:
@josemigallas the comment does it for strings, not for arrays 😞

Reply:
Ugh! OK let's "commit suggestion" it, no need for a push.